viewer/src/index.ts

```diff
@@ -1,6 +1,10 @@
 /*!
  * Copyright 2020 Cognite AS
  */
+declare let VERSION: string;
+const globalProps = {
```

Comment:
isn't it unused here? Not sure why linter allowed to commit it...

Reply:
Created a PR specifically for such cases https://github.com/cognitedata/reveal/pull/739